### ISSUE TITLE
Tests are failing

### ISSUE THREAD

**Issue by soxofaan** [danielfrg/pelican-jupyter#135]
I'm trying to run the tests and they seem to be failing.
For example see the github actions runs in my fork: https://github.com/soxofaan/pelican-jupyter/actions/workflows/test.yml

It seems there are various compatibility issues with dependencies

**Comment by danielfrg** [comment]
Yes, sadly I am not using this library anymore personally so i dont have time to update fix those issues.

**Comment by soxofaan** [comment]
Ok, no problem

Should https://github.com/danielfrg/pelican-jupyter be considered "unmaintained", or would you still accept a PR if I manage to figure this out?

**Comment by danielfrg** [comment]
I totally accept PRs and make releases with those when I have time but I think it should be considered unmaintened.

I am happy to add you to it if you want to be a mantainer!

**Comment by soxofaan** [comment]
> I am happy to add you to it if you want to be a mantainer!

@danielfrg  thanks for offering that. Unfortunately, to be honest, I also have very limited bandwidth to take up the role/responsibility of maintaining this package I'm afraid. I only use it for a small, low frequency side project. 

I think I fixed the broken test issue in #136 by the way
